central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -134,6 +133,20 @@ func (suite *PLOPDataStoreTestSuite) TestPLOPAdd() {
 		},
 	}
 
+	for _, indicator := range indicators {
```

Comment:
Multiple events with no matching process indicators. Two consecutive open and two consecutive close. Enable unit tests in CI.

Reply:
Done